<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLayerDialog</class>
 <widget class="QDialog" name="CLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>134</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>整体</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>开始值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startValueEdit">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>量程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valueRangeEdit">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>刻度数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scountEdit">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>96</width>
     <height>158</height>
    </rect>
   </property>
   <property name="title">
    <string>位置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string> 左</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leftEdit">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> 顶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="topEdit">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string> 宽</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="widEdit">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string> 高</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="heightEdit">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>134</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>直表参数</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QRadioButton" name="horRadioBtn">
        <property name="text">
         <string> 水平</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="vertRadioBtn">
        <property name="text">
         <string> 垂直</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>136</width>
     <height>92</height>
    </rect>
   </property>
   <property name="title">
    <string>圆表参数</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>开始角度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startAngleEdit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>量程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AngleRangeEdit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
